### ISSUE TITLE
Hashing not working for certain url cases

### ISSUE THREAD

**Issue by madishka** [alvarotrigo/fullPage.js#1893]
Hey there,
I have a a website that has two different flows depending on if a person is the first user creating a "team", or subsequent users joining that "team". For purposes of clarity, I will call them "captain" and "member". 

For both flow's I am using fullpage.js on their "index" pages.

The captain's flow url is as follows: `http://www.website.com/` and when it pages it works like `http://www.website.com/#anchor`
It works great in the captain's flow, and as expected.

In the member's flow, the url is as follows `http://www.website.com/invite-code/someinvitecode==` where `someinvitecode` represents a series of letters and numbers, and has the == at the end as it is an encoded GUID. When I scroll through the anchors here I get `http://www.website.com/invite-code/someinvitecode==#/anchor` and if I try reworking it to be `http://www.website.com/invite-code/someinvitecode==/#anchor` it corrects to `http://www.website.com/invite-code/someinvitecode==/#/anchor`

The markup for both pages is identical, I have pasted each of them below. I am curious to know why it is adding the extraneous `/` and don't know where to start.

CAPTAINS MARK UP

```
<div id="fullpage">
		<div data-anchor="firstPage" class="section" id="section0">
			FIRST PAGE
		</div>
		<div data-anchor="secondPage" class="section" id="section1">
			SECOND PAGE
		</div>
	</div>

	<script>
		$(document).ready(function() {
		  $('#fullpage').fullpage({
		    responsiveWidth: 992,
		    anchors: ['firstPage', 'secondPage']
		  }); 
		});
	</script>
```

MEMBER MARK UP
```
  	<div id="fullpagemember">
		<div data-anchor="sectionOne" class="section" id="section0">
			FIRST PAGE
		</div>
		<div data-anchor="sectionTwo" class="section" id="section1">
			SECOND PAGE
		</div>
	</div>

	<script>
		$(document).ready(function() {
		  $('#fullpagemember').fullpage({
		    responsiveWidth: 992,
		    anchors: ['sectionOne', 'sectionTwo']
		  }); 
		});
	</script>
```

For both, the paging works create, it's just that when I try to scroll between sections through a CTA, it is not working properly in the members version. I am referencing it through a button like so:
`<a href="#sectionTwo" class="cta__button--white">BUTTON TEXT</a> `

Thanks for all of your help.

**Comment by alvarotrigo** [comment]
> . When I scroll through the anchors here I get http://www.website.com/invite-code/someinvitecode==#/anchor

That doesn't make much sense.
Fullpage.js only add `/` when you are using sections anchors AND slide anchors.
In your case, that seems to be an slide anchor and **an empty section anchor**.

Which section anchors are you using? Which slide anchors? Can you add a real example URL?

**Comment by alvarotrigo** [comment]
Can you provide a link with the reproduction?

**Comment by alvarotrigo** [comment]
> it's just that when I try to scroll between sections through a CTA,

What is a CTA?

**Comment by madishka** [comment]
> What is a CTA?

Call-to-action, a button.

> Is anchor the slide anchor then? Or the section one?

The anchor **should** be a section anchor.. but for some reason it is acting as a slide anchor, according to your js link you sent. I am unsure why it is doing this? I have copied my mark up into a jsfiddle and it works as expected.. scrolling between sections. I am going to try reducing my mark up down, and it appears that it either has to do with my project url in this case, or it is in conflict with the other fullpage() function I have in another file? Is this something that you have seen before? I cannot send the link for the project directly under NDA, but I can try to create a github.io with a reduced markdown so you can fiddle with it as jsfiddle produces it correctly.

**Comment by alvarotrigo** [comment]
> Is this something that you have seen before?

Nop, first time.

> I cannot send the link for the project directly under NDA, but I can try to create a github.io with a reduced markdown so you can fiddle with it as jsfiddle produces it correctly.

A reproduction in jsfiddle or codepen would be ideal.
But as I said, it seems that somehow your section anchor is being omitted. Can you inspect your page DOM structure and find the `data-anchor` attribute in the section?

**Comment by alvarotrigo** [comment]
Make sure you are not overriding the fullpage.js `anchors` options with a custom `data-anchor` tag in the section before initialization.

**Comment by madishka** [comment]
I have solved this issue! I was becoming increasingly frustrated because when I put my mark up into the JSFiddle, it worked exactly as it should.. so I copied that code into my project and it no longer worked again!

Turns out, I likely had some invisible characters from when I was copy/pasting in my JS file for the controller I am using on this page in Angular. I debugged it down to a $http declaration in my controller to be the issue, and once I removed it, and readded it, the entire page worked as the captain's flow.

Thanks a lot, love the library. Once the project goes live I will be sure to send you a follow up.
